_objects.layout.scss

```diff
@@ -85,6 +107,11 @@ $_supercell-spacing-unit-large: round($supercell-spacing-unit * 2) !default;
     font-size: 0;
   }
 
+  &.o-layout--balanced {
```

Comment:
Hmm not too fond of using an `&` here when all of our other modifiers are flat - could we not wrap all of these styles for `o-layout` and `o-layout__item` in a modifier at the bottom?

Reply:
Sure! I'll take a look, totally missed these comments.